enterprise/backend/src/metabase_enterprise/representations/toucan/core.clj

```diff
@@ -0,0 +1,68 @@
+(ns metabase-enterprise.representations.toucan.core
+  (:require
+   [metabase.api.common :as api]
+   [metabase.config.core :as config]))
+
+(defmulti with-toucan-defaults
+  "Adds default, required values to a toucan representation."
+  {:arglists '([t2-model toucan-entity])}
```

Comment:
Canonically we call these things "instances", e.g. "an instance of a :model/Card", see for example `t2/instance`. Please use that instead of "entity"